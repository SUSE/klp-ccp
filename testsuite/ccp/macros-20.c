#define FOO()

#include "macros-20.h"

void pu_f(void)
{
	G();
}
