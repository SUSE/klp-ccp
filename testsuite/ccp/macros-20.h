#if defined(G)

#elif defined(FOO)

#define G g

#define EXTERN_VOID extern void

EXTERN_VOID G(void)
{}

#endif
