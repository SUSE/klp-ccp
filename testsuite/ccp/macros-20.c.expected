#define FOO()

#if defined(G)
#error "klp-ccp: non-taken branch"
#elif defined(FOO)

#define G g

static void G(void);

#endif
#undef G
#include "macros-20.h"

void klpp_pu_f(void)
{
	G();
}
